p02/p02b.jq

```diff
@@ -1,17 +1,22 @@
-#!/usr/bin/env jq -s -R -f
+#!/usr/bin/env jq --raw-input -f
```

Comment:
1. Ahh okay. `+=` is nice. I also will have to try out `error` and `debug`

2. Is slurping and then splitting on "\n" redundant?

3. [jq docs](https://stedolan.github.io/jq/manual/#ConditionalsandComparisons) say `if A then B end is the same as if A then B else . end. That is, the else branch is optional, and if absent is the same as .`  But if I try omitting an else, it gives me a compile error